com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDShadowContext.hlsl

```diff
@@ -19,8 +19,7 @@ struct HDShadowContext
 
 TEXTURE2D(_ShadowmapAtlas);
 TEXTURE2D(_ShadowmapCascadeAtlas);
-TEXTURE2D(_AreaShadowmapAtlas);
-TEXTURE2D(_AreaShadowmapMomentAtlas);
```

Comment:
Uhm, while I am generally agreeing with this, I wonder if this is a problem if we ever need both the moment and the depth based one. Probably not, but I'd add a comment here saying that it contains moments if EVSM is used

Reply:
I had the feeling that all we cared about here was just to have the final usable shadow map texture so we did not want to know about algorithm specific information.
I can still add a comment but I feel that it's sort of a "leak" of implementation detail into something that should be agnostic.

Reply:
Yeah you are right. I agree  here, just slightly worried that this might get accessed as traditional depth-map by the unknowing, but I guess is over worrying.